apps/bare-expo/ios/Pods/Headers/Private/EXSQLite/EXSQLite.h

```diff
@@ -1 +1 @@
-../../../../../../../packages/expo-sqlite/ios/EXSQLite/EXSQLite.h
\ No newline at end of file
+../../../../../../../packages/expo-sqlite/ios/EXSQLite/EXSQLite.h
```

Comment:
There's no difference and the diff is still showing. 🤔 Maybe squashing offending commits would help remove this entry?